dashboard/app/models/level.rb

```diff
@@ -239,6 +239,14 @@ def strip_name
     self.name = name.to_s.strip unless name.nil?
   end
 
+  # Find the first script this level belongs to that is a HOC script.
+  # Returns nil if this level does not belong to any HOC scripts.
+  def find_hoc_script
+    hoc_script_level = self.script_levels.find{|l| l.script.hoc?}
```

Comment:
After meeting with Charing and Alice, we decided to always link the "Make my own app" menu item to https://code.org/learn. I'll remove this function.